package.json

```diff
@@ -107,18 +102,19 @@
         "rollup-plugin-istanbul": "^3.0.0",
         "rollup-plugin-terser": "^7.0.2",
         "size-limit": "^7.0.8",
-        "typedoc": "^0.23.10",
-        "typedoc-plugin-markdown": "^3.13.4",
         "typescript": "^4.7.4",
-        "vue-tabs-component": "^1.5.0",
-        "vuepress": "^1.9.7",
-        "vuepress-plugin-code-copy": "^1.0.6",
-        "vuepress-plugin-flexsearch": "^0.3.0",
-        "vuepress-plugin-redirect": "^1.2.5",
-        "vuepress-plugin-tabs": "^0.3.0",
-        "vuepress-plugin-typedoc": "^0.11.0",
-        "vuepress-theme-chartjs": "^0.2.0",
-        "webpack": "^4.46.0",
         "yargs": "^17.0.1"
+    },
+    "engines": {
+        "pnpm": "^7.0.0"
+    },
+    "packageManager": "pnpm@7.9.0",
+    "pnpm": {
```

Comment:
Could you explain why these configs are needed?

Reply:
pnpm enforces `peerDependencies` more strictly than npm does. Some of the Chart.js utility packages specify Chart.js 3 as a peer dependency and that check fails since we're on Chart.js 4 now